addon/-private/system/store.js

```diff
@@ -2265,48 +2272,52 @@ Store = Service.extend({
 
     @method _push
     @private
-    @param {Object} data
+    @param {Object} apiDoc
```

Comment:
I don't immediately grok what `apiDoc` means here, is this a common term? Maybe you could add a short description after the name to make it clearer to folks?